### PR TITLE
fix issue (#7152 : Improve relation empty states on record page) 

### PR DESCRIPTION
**Pull request by falgunmpatel**
- "No xxx" removed for empty relations
- All(0) removed

### CODE REVIEW

**Comment by Bonapara** [comment]
Thanks @falgunmpatel 

- The gap below the relation name should match the gap above it. (Make sure the spacing remains as today when there are some relations under though 🚨)
- You also missed the point about showing the CTA when there is no relation displayed

**Comment by falgunmpatel** [comment]
Thanks for your review @Bonapara . I will keep these points in my mind while solving any issues. Also I will rectify my mistakes in this issue soon.

**Comment by falgunmpatel** [comment]
> Thanks for your review @Bonapara . I will keep these points in my mind while solving any issues. Also I will rectify my mistakes in this issue soon.

Also should I delete the component that I commented.

File: packages/twenty-front/src/modules/object-record/record-show/record-detail-section/components/RecordDetailRelationRecordsListEmptyState.tsx [hunk 1]
```diff
@@ -1,35 +1,37 @@
...
+//   const { getIcon } = useIcons();
+//   const Icon = getIcon(relationObjectMetadataItem.icon);
+
+//   return (
+//     <StyledRelationRecordsListEmptyState>
+//       <Icon size={theme.icon.size.sm} />
+//       <div>No {relationObjectMetadataItem.labelSingular}</div>
+//     </StyledRelationRecordsListEmptyState>
+//   );
+// };
```
Comment (bosiraphael) [inline packages/twenty-front/src/modules/object-record/record-show/record-detail-section/components/RecordDetailRelationRecordsListEmptyState.tsx:37]: You can remove this component :)

**Review by bosiraphael — COMMENTED** [review]
Hello @falgunmpatel ! Thank you for your contribution :) LGTM, just remove the unused component

**Comment by Bonapara** [comment]
Thanks @falgunmpatel! Could you adjust the skeleton loading by removing the relations skeleton? It causes too much flashing since we don't know if they will have content or not.

# Before

image

# Desired

image

**Comment by falgunmpatel** [comment]
@Bonapara I have removed skeletons causing flashing effect and achieved desired output.

File: packages/twenty-front/src/modules/object-record/record-show/record-detail-section/components/RecordDetailRelationSection.tsx [hunk 2]
```diff
@@ -158,6 +146,8 @@ export const RecordDetailRelationSection = ({
       recordId,
     });
 
+  if (!loading) return null;
```
Comment (bosiraphael) [inline packages/twenty-front/src/modules/object-record/record-show/record-detail-section/components/RecordDetailRelationSection.tsx:149]:
```suggestion
  if (loading) return null;
```

**Review by bosiraphael — COMMENTED** [review]
Wrong condition, otherwise LGTM ;) Thank you for your contribution

**Review by bosiraphael — APPROVED** [review]
LGTM

**Pull request merged**